testgrid config check failing for `ci-test-infra-resultstore-upload`

This is blocking a PR for esp-v2: https://github.com/GoogleCloudPlatform/oss-test-infra/pull/317

```
==================== Test output for //config/tests/testgrids:go_default_test:
--- FAIL: TestKubernetesProwInstanceJobsMustHaveMatchingTestgridEntries (3.20s)
    config_test.go:436: Job ci-test-infra-resultstore-upload does not have a matching testgrid testgroup
```

This was working earlier in the day. I don't know how it started failing, as there were no other PRs (other than esp-v2).

--- a/prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml
+++ b/prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml
@@ -197,7 +197,7 @@ periodics:
       secret:
         secretName: oauth-token
 - interval: 5m
-  name: ci-test-infra-resultstore-upload
+  name: ci-google-oss-test-infra-resultstore-upload
   annotations:
     testgrid-dashboards: googleoss-test-infra
     testgrid-tab-name: resultstore

